doc/api/fs.md

```diff
@@ -3077,6 +3077,74 @@ error raised if the file is not available.
 To check if a file exists without manipulating it afterwards, [`fs.access()`]
 is recommended.
 
+For example, given the following folder structure:
+
+-txtDir
+--file.txt
+-app.js
+
+The next program will check for the stats of the given paths:
+
+```js
+const fs = require('fs');
+
+const pathsToCheck = ['./txtDir', './txtDir/file.txt'];
+
+for (i = 0; i < pathsToCheck.length; i++) {
```

Comment:
To make this run in strict mode:
```suggestion
for (let i = 0; i < pathsToCheck.length; i++) {
```

Reply:
I'm a bit unsure if the push was done correctly but this should've been fixed in the latest commit.